docs/docs/plugins.md

```diff
@@ -80,6 +80,29 @@ module.exports = {
 
 Note that plugin options will be stringified by Gatsby, so they cannot be functions.
 
```

Comment:
I think adding a header at this point would be helpful. What about something like ## Loading plugins from your local plugins folder

Reply:
Yes you are right. Added the heading!